ingestion-beam/src/test/java/com/mozilla/telemetry/heka/HekaIOTest.java

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It's not great to move files in a PR while making changes, as it makes it quite difficult to review what actually changed. I don't think it's that big a deal in this case, but something to bear in mind for the future.